cmd/analyse.go

```diff
@@ -49,23 +50,26 @@ func destructor(cmd *cobra.Command, args []string) {
 				// If file doesn't exists, continue
 				continue
 			}
+			return err
 		}
 		e := os.Remove(file)
 		if e != nil {
-			log.Fatal().Msgf("Error clearing files %s", file)
+			return fmt.Errorf("error clearing files %s", file)
 		}
 	}
+	return nil
 }
 
 //runAnalyse is controller func for analyses cmd.
-func runAnalyse(cmd *cobra.Command, args []string) {
+func runAnalyse(cmd *cobra.Command, args []string) error {
+	telemetry.SetFlag(cmd.Context(), "json", jsonOut)
+	telemetry.SetFlag(cmd.Context(), "verbose", verboseOut)
 	if !viper.IsSet("crda_key") {
-		fmt.Fprintln(os.Stdout,
-			color.RedString("\u2718 "),
-			"Please run `crda auth` command first.",
-		)
-		os.Exit(1)
+		telemetry.SetExitCode(cmd.Context(), 1)
+		return errors.New(
+			"please run `crda auth` command first")
 	}
+	telemetry.SetCrdaKey(cmd.Context(), viper.GetString("crda_key"))
```

Comment:
I'm not sure whether we can send the CRDA key as is.
CC: @prashbnair

Reply:
Is is being sent in VS code telemetry?

Reply:
nope, this is not sent.

Reply:
What's the final verdict..? @arajkumar @prashbnair

Reply:
Dont send it is what I think

Reply:
Ack.